iso15118/shared/utils.py

```diff
@@ -1,10 +1,64 @@
 import asyncio
 import logging
-from typing import Coroutine, List
+from typing import Coroutine, List, Optional
+
+from iso15118.shared.exceptions import NoSupportedEnergyServices, NoSupportedProtocols
+from iso15118.shared.messages.enums import Protocol, ServiceV20
 
 logger = logging.getLogger(__name__)
 
 
+def _format_list(read_settings: List[str]) -> List[str]:
+    read_settings = list(filter(None, read_settings))
+    read_settings = [setting.strip().upper() for setting in read_settings]
+    read_settings = list(set(read_settings))
+    return read_settings
+
+
```

Comment:
As this is in shared, the one in secc_settings.py could perhaps be removed?
https://github.com/SwitchEV/iso15118/blob/2b146b483996863e82fca111db33dabf39f293fc/iso15118/secc/secc_settings.py#L19